django-stubs/contrib/postgres/forms/jsonb.pyi

```diff
@@ -1,8 +1,9 @@
 from django import forms
-from typing import Any
+from typing import Any, Type, Union
 
 class InvalidJSONInput(str): ...
 class JSONString(str): ...
 
 class JSONField(forms.CharField):
+    widget: Union[forms.Widget, Type[forms.Widget]] = ...
```

Comment:
Same as in the comment about `field`

Reply:
Django is weird ... the field classes expect `widget` to define a static attribute specify the widget class, but then instantiate and set the self.widget to the instance 🤦‍♂ ... so if I pull the above line to `forms.Field` I start seeing errors like this:
```
django-sources/tests/postgres_tests/test_ranges.py:723: error: Argument 1 to "render" of "Widget" has incompatible type "str"; expected "Widget"
django-sources/tests/postgres_tests/test_ranges.py:723: error: Too few arguments for "render" of "Widget"
django-sources/tests/postgres_tests/test_ranges.py:727: error: Argument 1 to "render" of "Widget" has incompatible type "str"; expected "Widget"
django-sources/tests/postgres_tests/test_ranges.py:727: error: Argument 2 to "render" of "Widget" has incompatible type "None"; expected "str"
```

I could only resolve this by pulling defining this in child class:

```
class BaseRangeField(forms.MultiValueField):
    widget: Widget
```

In that case, I guess it's better to keep the `Union` on the parent, and add the `Argument 1 to "render" of "Widget" has incompatible type "str"` error under: `enabled_test_modules.py`?